cleverhans/attacks_tf.py

```diff
@@ -583,23 +583,27 @@ def jacobian_augmentation(sess, x, X_sub_prev, Y_sub, grads, lmbda,
     # Create new numpy array for adversary training data
     # with twice as many components on the first dimension.
     X_sub = np.vstack([X_sub_prev, X_sub_prev])
-
-    # For each input in the previous' substitute training iteration
-    for ind, prev_input in enumerate(X_sub_prev):
-        # Select gradient corresponding to the label predicted by the oracle
-        grad = grads[Y_sub[ind]]
-
-        # Prepare feeding dictionary
-        feed_dict = {x: np.reshape(prev_input, input_shape)}
+    nsamples = X_sub_prev.shape[0]
```

Comment:
Could you change the name of the var to `num_samples` for consistency with the rest of the library?